### DISCUSSION TITLE
High CPU Usage in Ubuntu Hetzner VM

### DISCUSSION THREAD

**Discussion by MohdRashid01** [community/community#45573] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Due to high usage of CPU in ubuntu vm it become unresponsive cannot login into VM and we have to restart direct through hetnzer stop and start option

Below is the screenshot which i capture and i dont know where is that file coming and why it usage is high any help would be great

./sdjdshdgdsdsfsfausjashsaggsafsfaa.x86

Even i try everything to locate,search and find the file but didnt get it

![image](https://user-images.githubusercontent.com/7812871/215181870-44883c6c-d076-40bd-9ac9-fda5cbc507ef.png)

**Comment by jge162** [comment]
@MohdRashid01 use the lsof command to identify the process that is using the file. To do this, you can use the command lsof -p <PID> to get information about the process, including the path to the file it is using. This should allow you to locate the file and determine why it is using so much CPU. You can then take the appropriate steps to address the issue, such as stopping or killing the process.

Do you have any questions GitHub related please lmk thanks.

**Reply by MohdRashid01** [reply]
i try like this lsof -p 379669 getting empty response and 
```
ubuntu@ubuntu:~# lsof -p 379669 ./sdjdshdgdsdsfsfausjashsagsafsfaa.x86
lsof: status error on ./sdjdshdgdsdsfsfausjashsagsafsfaa.x86: No such file or directory
```
OR
```
lsof -p 379669
```

**Reply by jge162** [reply]
If you're getting an empty response, it could mean a few things->

- The process with PID 379669 doesn't exist or has already terminated.
- The process is not holding any open files.

**Reply by jge162** [reply]
Try using the "ps" command to check if the process with PID 379669 is running, and if it is, check if it has any open files using the "lsof" command without the PID argument
